### PR TITLE
fixing 006-010 and add new 011-015

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@LukasReschke credits have been updated

File: advisories/advisories.rss [hunk 1]
```diff
@@ -6,31 +6,31 @@
  <description>The ownCloud security advisories as a RSS feed</description>
  <ttl>1800</ttl><item>
   <title>Server: Read-only share recipient can restore old versions of file (oC-SA-2016-015)</title>
-  <description>&lt;br/&gt;&lt;hr/&gt;&lt;p&gt;&lt;strong&gt;&lt;a href=&quot;https://owncloud.org/security/advisory/?id=oC-SA-2016-015&quot;&gt;For more information please consult the official advisory.&lt;/a&gt;&lt;/strong&gt;&lt;/p&gt;</description>
+  <description>&lt;p&gt;The restore capability of Nextcloud was not verifying whether an user has only read-only access to a share. Thus an user with read-only access was able to restore old versions.&lt;/p&gt;&lt;br/&gt;&lt;hr/&gt;&lt;p&gt;&lt;strong&gt;&lt;a href=&quot;https://owncloud.org/security/advisory/?id=oC-SA-2016-015&quot;&gt;For more information please consult the official advisory.&lt;/a&gt;&lt;/strong&gt;&lt;/p&gt;</description>
```
Comment (danimo) [inline advisories/advisories.rss]: ownCloud

File: advisories/advisories.rss [hunk 2]
```diff
@@ -6,31 +6,31 @@
...
+  <description>Due to a recent migration of the Gallery app to the new sharing endpoint a parameter changed from an integer to a string value. This value wasn't sanitized before and was thus now vulnerable to a Cross-Site-Scripting attack.&lt;/p&gt;&lt;p&gt;To exploit this vulnerability an authenticated attacker has to share a folder with someone else, get them to open the shared folder in the Gallery app and open the sharing window there.&lt;/p&gt;&lt;p&gt;Since Nextcloud employes a strict Content-Security-Policy this vulnerability is only exploitable in browsers not supporting Content-Security-Policy. You can check at &lt;a href=&quot;http://caniuse.com/#feat=contentsecuritypolicy&quot;&gt;caniuse.com&lt;/a&gt; whether your browser supports CSP.&lt;br/&gt;&lt;hr/&gt;&lt;p&gt;&lt;strong&gt;&lt;a href=&quot;https://owncloud.org/security/advisory/?id=oC-SA-2016-011&quot;&gt;For more information please consult the official advisory.&lt;/a&gt;&lt;/strong&gt;&lt;/p&gt;</description>
```
Comment (danimo) [inline advisories/advisories.rss]: ownCloud

File: advisories/oc-sa-2016-011.php [hunk 3]
```diff
@@ -16,7 +16,9 @@
         <p>CWE: <a href="https://cwe.mitre.org/data/definitions/79.html">Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting') (CWE-79)</a></p>
 <p>HackerOne report: <a href="https://hackerone.com/reports/145355">145355</a></p>
         <h3>Description</h3>
-        <p></p>
+        <p>Due to a recent migration of the Gallery app to the new sharing endpoint a parameter changed from an integer to a string value. This value wasn't sanitized before and was thus now vulnerable to a Cross-Site-Scripting attack.</p>
+<p>To exploit this vulnerability an authenticated attacker has to share a folder with someone else, get them to open the shared folder in the Gallery app and open the sharing window there.</p>
+<p>Since Nextcloud employes a strict Content-Security-Policy this vulnerability is only exploitable in browsers not supporting Content-Security-Policy. You can check at <a href="http://caniuse.com/#feat=contentsecuritypolicy">caniuse.com</a> whether your browser supports CSP.</p>
```
Comment (danimo) [inline advisories/oc-sa-2016-011.php]: ownCloud

File: advisories/oc-sa-2016-015.php [hunk 4]
```diff
@@ -16,7 +16,8 @@
         <p>CWE: <a href="https://cwe.mitre.org/data/definitions/275.html">Permission Issues (CWE-275)</a></p>
 <p>HackerOne report: <a href="https://hackerone.com/reports/146067">146067</a></p>
         <h3>Description</h3>
-        <p></p>
+        <p><p>The restore capability of Nextcloud was not verifying whether an user has only read-only access to a share. Thus an user with read-only access was able to restore old versions.</p>
```
Comment (danimo) [inline advisories/oc-sa-2016-015.php]: ownCloud

**Pull request merged**